### ISSUE TITLE
eqMac cannot install driver

### ISSUE THREAD

**Issue by EasyG0ing1** [bitgapp/eqMac#291]
I'm running Catalina 10.15.4. Downloaded the .DMG, mounted it, dropped the app in the app folder, ran it, gave it my password but after trying to install the driver for a few seconds, the window goes away although the icon in the dock bar is still lit, so when CMD+TAB back to it, it shows this:

<img width="421" alt="Screen Shot 2020-06-01 at 11 10 44 AM" src="https://user-images.githubusercontent.com/10106834/83439851-c32dd880-a3f8-11ea-9e7a-c4ad3fbc491e.png">

So first, I shut down all running apps and tried again without success, then I rebooted and tried it first thing after login without success... then I ran the terminal uninstall command you have posted on the web site ... that didn't work either.

Is there a more direct way to get the driver installed? Like a sudo method perhaps?

Thank you

**Comment by nodeful** [comment]
Try `brew cask install eqmac`

**Comment by erikhuizinga** [comment]
I have the same issue.
I tried uninstalling eqmac (remove from applications, run cleanup command in terminal), then installing the latest (v0.3.0) version manually from GitHub. Homebrew cask still only knows about v0.2.0, so I didn't try that. MacOS 10.14.6 (18G5033) here.

**Comment by nodeful** [comment]
Can you please read through #317 and try some of the steps the user went through there. Make sure to use v0.3.1 when you try.

**Comment by erikhuizinga** [comment]
Restarting my mac.... oof, that's a big step to ask from a pro user like me! ;)

JK. I'll try that later, and the other steps described! Thanks for the response.

**Comment by nodeful** [comment]
Haha! I know it's not ideal. The installation process is trying it's best to not have to restart, but unfortunately this process is not very predictable :(

**Comment by erikhuizinga** [comment]
Just rebooted, still cannot install driver. I'm not sure what steps I can take from #317, because it seems that that user does have eqMac running successfully, but had troubles _upgrading_ the driver. I cannot even open eqMac from a clean installation without the driver install failure.

**Comment by nodeful** [comment]
Try running these commands in Terminal:
`sudo cp -f -r /Applications/eqMac.app/Contents/Resources/eqMac.driver /Library/Audio/Plug-Ins/HAL/`
`sudo launchctl kickstart -kp system/com.apple.audio.coreaudiod` or just restart your Mac

**Comment by NaturallyAsh** [comment]
@nodeful Thanks man! sudo worked for me! Catalina 10.15.5.

**Comment by iamtodor** [comment]
@nodeful 
macOS version 10.15.7
download the latest available `eqMac.pkg` from https://eqmac.app/
I've got the same dialog as @EasyG0ing1 

Unfortunately, the abovementioned commands did not help me:
```
sudo cp -f -r /Applications/eqMac.app/Contents/Resources/eqMac.driver /Library/Audio/Plug-Ins/HAL/
cp: /Applications/eqMac.app/Contents/Resources/eqMac.driver: No such file or directory
```

Is there anything I am missing or I can provide to you for father investigation and resolution?

**Comment by nodeful** [comment]
Thanks for being patient with me.
I have just released v1.0.0. You can download it here: https://github.com/bitgapp/eqMac/releases/tag/v1.0.0
Or from the website: https://eqmac.app
Could you please let me know if this issue is still present. 
Thanks again.

Roman

**Comment by erikhuizinga** [comment]
Upgrading from v? to v1.0.0 went smooth for me. Just in case: I did restart my mac when the installer prompted me to do so, and I closed all apps before proceeding with the installation. Please consider closing this issue.

**Comment by plef** [comment]
Had same issue with the latest release, so I followed manual uninstall instructions as seen on project homepage FAQ section, still with no success. But eventually I realised that I still had the development version installed by Xcode Debug run, which apparently conflicted release bundle driver installation. So I completely wiped out all the installed artefacts produced by Xcode Build & Run finally, the fresh installation completed successfully, including the driver.

So if somebody ever tried Xcode to run development version, and forgot about that alter on, wiping it out should help.

Other cases might include permissions, etc., not my case, so I leave others to advise on here.
